src/main/java/nextstep/subway/line/dto/LineResponse.java

```diff
@@ -1,29 +1,39 @@
 package nextstep.subway.line.dto;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import nextstep.subway.line.domain.Line;
+import nextstep.subway.station.domain.Station;
+import nextstep.subway.station.dto.StationResponse;
 
 import java.time.LocalDateTime;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
 
 public class LineResponse {
     private Long id;
     private String name;
     private String color;
+    @JsonProperty("stations")
+    private List<Station> stations;
```

Comment:
`LineResponse` 객체에서 `Station` 엔티티를 직접적으로 가지고있기보다 특정 dto(response)로 포장한 객체를 사용해보면 어떨까요?
ex) `StationResponse`
```suggestion
    private List<StationResponse> stations;
```

외부에 도메인(엔티티)를 노출하지 않는게 좋을 것 같습니다~